config/locales/interface/input_elements/en_demand_buildings.yml

```diff
@@ -473,3 +473,37 @@ en:
         switches to hydrogen. The threshold value can be adjusted in the section 
         <a href=/scenario/flexibility/flexibility_net_load/demand-response-behavior-of-hybrid-heat-pumps
         >Demand response - behavior of hybrid heat pumps</a>.
+    buildings_space_heater_hybrid_crude_oil_heatpump_air_water_electricity_share:
+      title: Hybrid air heat pump (oil)
+      short_description:
+      description: |
+        This heater is a combination of an electric heat pump that draws its heat 
+        from the outside air and an efficient conventional oil-fired boiler. <br/><br/>
+        This works as follows: <br/> 
+        The oil-fired part takes over all of the heating of the building on cold 
+        winter days, when heat demand peaks. Because of this clever trick, the heat 
+        pump has less impact on the electricity grid than an all-electric heat pump does. 
+        Smart use of hybrid crude oil heat pumps may therefore avoid the need to reinforce 
+        local power grids. <br/><br/>
```

Comment:
Referring to the behaviour of the HHP as a "clever trick" does not seem very professional.